engine.js

```diff
@@ -116,8 +116,12 @@ module.exports = function (options) {
         };
 
         // parentheses are only needed when a scope is present
-        var scope = answers.scope.trim();
-        scope = scope ? '(' + answers.scope.trim() + ')' : '';
+        var scope = answers.scope;
+        if (Array.isArray(scope)) {
+          scope = scope.join(', ');
+        }
+        scope = scope.trim();
+        scope = scope ? '(' + scope + ')' : '';
```

Comment:
That is fine. But maybe `scope.length` makes more clear the intention of this ternary.

Reply:
I'm going by applying the least amount of changes to the already existing library in order to prevent a huge mismatch between this and the base fork.
If I am to go with the boy-scout rule here, there will be a lot more changes throughout the file.
My question would be, is it really necessary?